example/single-jvm-example/single-jvm-example-client/src/main/java/uk/gov/gchq/palisade/example/client/ExampleSimpleClient.java

```diff
@@ -91,27 +75,28 @@ protected PolicyService createPolicyService() {
                                 )
                 )
         );
-        return policyService;
-    }
 
-    @Override
-    protected ResourceService createResourceService() {
-        final ResourceService resourceService = super.createResourceService();
-        resourceService.addResource(new AddResourceRequest(
-                new DirectoryResource("dir1", RESOURCE_TYPE),
-                new FileResource("file1", RESOURCE_TYPE),
-                new SimpleConnectionDetail()
-        ));
+        // The sys admin needs to add the resources
         resourceService.addResource(new AddResourceRequest(
                 new DirectoryResource("dir1", RESOURCE_TYPE),
                 new FileResource("file1", RESOURCE_TYPE),
-                new SimpleConnectionDetail()
+                new SimpleConnectionDetail(new SimpleDataService(palisadeService, new ExampleSimpleDataReader()))
         ));
-        return resourceService;
+
+        // Wait for the users, policies and resources to be loaded
+        try {
+            Thread.sleep(1000);
```

Comment:
Sorry, I'm being a pedant here, please could the sleep be replaced with a CompleteableFuture that calls waits on these requests to complete? CompletableFuture.allOf(.....)